ui/Dockerfile

```diff
@@ -5,7 +5,7 @@ WORKDIR /app
 RUN apk add --update nodejs npm
 RUN apk add --update apache2-utils
 RUN apk add --update libwebp=1.3.2-r0
-RUN apk add --update libx11=1.8.7-r0
+#RUN apk add --update libx11=1.8.7-r0
```

Comment:
Just remove it completely if it's not needed. Also check if libwebp is needed and remove it completely too or at least remove package version.

Reply:
Also please use `npm ci` instead of `npm install` because it installs exact versions from the lock file and will not suddenly break because some dependency was updated.

Reply:
@x1m3 Still libwebp is there with strict version (which could already be removed from the package repos). And npm install is also there.